apps/coscrad-frontend-e2e/src/e2e/resources/photographs/queries/photograph.detail.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,266 @@
+import { AggregateType, LanguageCode, MIMEType } from '@coscrad/api-interfaces';
+import {
+    buildDummyAggregateCompositeIdentifier,
+    buildDummyUuid,
+} from '../../../../support/utilities';
+import { seedDummyMediaItem } from '../../../shared/seed-dummy-media-item.cy';
+
+const photographAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.photograph,
+    2
+);
+
+const photographAggregateCompositeIdentifierNoNotes = buildDummyAggregateCompositeIdentifier(
+    AggregateType.photograph,
+    3
+);
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    12
+);
+
+const mediaItemCompositeIdentifierNoNotes = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    13
+);
+
+const mediaItemCompositeIdentifierNoConnections = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    14
+);
+
+const buildRoute = (id: string) => `/Resources/Photographs/${id}`;
+
+const photographTitle = 'My Photo';
+
+const photographTitleNoNotes = 'My Photo without any notes';
+
+const dummyPhotographer = 'Slank Mortimer';
+
+const languageCodeForTitle = LanguageCode.Haida;
+
+const validPhotographDetailRoute = buildRoute(photographAggregateCompositeIdentifier.id);
+
+const noteText = 'This is a note about a photograph';
+
+const contributors = {
+    creator: {
+        id: buildDummyUuid(112),
+        firstName: 'Jamethon',
+    },
+};
+
+describe(`the photograph detail page`, () => {
+    describe(`when the photograph with the given ID 123 does not exist`, () => {
+        beforeEach(() => {
+            cy.visit(buildRoute(`123`));
+        });
+
+        it(`should display the not found page`, () => {
+            cy.getByDataAttribute('not-found');
+        });
+    });
+
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(200);
+
+        cy.seedDataWithCommand(`CREATE_CONTRIBUTOR`, {
+            aggregateCompositeIdentifier: {
+                type: AggregateType.contributor,
+                id: contributors.creator.id,
+            },
+            shortBio: 'This is a hard-working language champion indeed!',
+            firstName: contributors.creator.firstName,
+            // we don't bother with the last name
```

Comment:
Is there a reason why?

Reply:
Not sure, this was there originally.

Reply:
I went ahead and added a last name. But I think what the comment should have been was that we accept the last name that is on the dummy date because seeing the first name show up on the screen is a good enough sanity check.